rust-toolchain.toml

```diff
@@ -1 +1 @@
-toolchains/stable/rust-toolchain.toml
\ No newline at end of file
+toolchains/nightly/rust-toolchain.toml
```

Comment:
Is this necessary?

Reply:
missed that! I reverted the change on the main branch